.vscode/launch.json

```diff
@@ -15,7 +15,7 @@
       "request": "attach",
       "protocol": "inspector",
       "name": "Attach to Process (Inspector Protocol)",
-      "port": 9229,
+      "port": 6004,
```

Comment:
I'll change this. This was just because the VF language server runs on port 6004.